encryption-decryption/src/com/nbugaenco/encryptdecrypt/Main.java

```diff
@@ -0,0 +1,119 @@
+package com.nbugaenco.encryptdecrypt;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class Main {
+
+    private static final Map<Parameter, String> params = new HashMap<>() {{
```

Comment:
A better option in my opinion would be to have static utility function which returns you this map, not to have it ``static`` in Main class